<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableItemForm</class>
 <widget class="QWidget" name="TableItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	color:rgb(255,255,255);
}</string>
  </property>
  <widget class="QLabel" name="lb_bookImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bookName</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookAuthor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bookAuthor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{	
	background-color:rgb(0, 0, 0);
	border-top-left-radius : 15px;

	border-bottom-right-radius : 15px;
}
QPushButton{

    
}

QPushButton:hover{


	border:1px solid blue;
	background-color: rgb(0,118,143);

}</string>
   </property>
   <property name="text">
    <string>详细信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
